src/net_processing.cpp

```diff
@@ -2446,7 +2446,6 @@ void PeerLogicValidation::ProcessMessage(CNode& pfrom, const std::string& msg_ty
 
             // Get recent addresses
             m_connman.PushMessage(&pfrom, CNetMsgMaker(nSendVersion).Make(NetMsgType::GETADDR));
-            pfrom.fGetAddr = true;
```

Comment:
This is unrelated to this PR, but could you replace this with a blank line instead of deleting the line. The `MarkAddressGood()` call below is unrelated to getting recent addresses, so should be visually separated from the line above.